db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -111,6 +100,13 @@ def vm_prov_option_value(prov_option, options_array = [])
   options_array
 end
 
+def requested_memory(args_hash, prov_type)
```

Comment:
@billfitzgerald0120 Can we change the prov_type to vendor, since prov_type would mean things like thick/thin/iso/pxe provisioning.